finite-supply.adoc

```diff
@@ -146,53 +146,53 @@ mempool—a small chance at a large amount can be worth more than a
 large chance at a small amount.
 ____
 
-To further put a wet blanket over our hopes for the future, if miners
-conduct fee sniping, it further incentivizes others to do the same,
+Throwing a wet blanket over our hopes for the future is the fact that if miners start
+conducting fee sniping, this will incentivize others to do the same,
 leaving even fewer honest miners. This could severely impair the
 overall security of Bitcoin. Harding goes on to list a few
-countermeasures that have been taken, for example using transaction
-lock time to restrict where in the blockchain the transaction may
+countermeasures that can be taken, such as relying on transaction
+time locks to restrict where in the blockchain the transaction may
 appear.
 
 So, given that the consensus on finite supply remains, the block
-subsidy will, thanks to
+subsidy will - thanks to
 https://github.com/bitcoin/bips/blob/master/bip-0042.mediawiki[BIP42]
-that fixed a very-long-term inflation bug ;), be zero from around year
-2140. Will the transaction fees after that be enough to secure the
-network? It's impossible to say, but we do know some things:
+which fixed a very-long-term inflation bug - get to zero around year
+2140. Will the transaction fees thereafter be enough to secure the
+network? It's impossible to say, but we do know a few things:
 
 * A century is a _long_ time from the Bitcoin perspective. If it is
-  still around, it has probably evolved enormously.
-* If an overwhelming economic majority finds it attractive to change
+  still around, it will have probably evolved enormously.
+* If an overwhelming economic majority finds it convenient to change
```

Comment:
"neccesary" instead of "convenient"? They would probably reluctantly do this out of necessity, rather than "happily" do it. Or maybe my understanding of the work convenient is flawed?

Reply:
I meant "convenient" in the sense of "useful toward a desired purpose" because this was my understanding of the sentence. Anyway, I think that "necessary" or "advantageous" work as well, you may pick whichever you prefer